scripts/live-connector-e2e/report.py

```
@@ -56,36 +96,165 @@ def load_results(results_dir: Path, *, roots: list[Path] | None = None) -> list[
+
+
+def load_candidate_regression_results(classifications: list[Classification]) -> list[dict]:
+    rows: list[dict] = []
+    for item in classifications:
+        if not item.is_candidate_regression:
+            continue
+        for row in item.result_rows:
+            # Classification metadata and row evidence must agree before a
+            # failure can enter candidate-regression content.
+            if _matches_candidate_failure(row, item):
+                rows.append(row)
```

Ruff (0.16.1)

[warning] Use `list.extend` to create a transformed list

(PERF401)